### PR TITLE
#MAN-502 Add LibGuides to person profile

### PR DESCRIPTION
**Pull request by cdoyle-temple**
Rework MAN-502 as it kept failing in Traivs.

### CODE REVIEW

File: db/migrate/20190926180617_add_slug_field_to_collections.rb [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
```
Comment (bibliotechy) [inline db/migrate/20190926180617_add_slug_field_to_collections.rb]: Why is this deleted?.

Comment (cdoyle-temple) [reply db/migrate/20190926180617_add_slug_field_to_collections.rb]: accident, inadvertant.

Comment (bibliotechy) [reply db/migrate/20190926180617_add_slug_field_to_collections.rb]: @cdoyle-temple ping

Comment (cdoyle-temple) [reply db/migrate/20190926180617_add_slug_field_to_collections.rb]: yah ahahha

**Comment by cdoyle-temple** [comment]
@bibliotechy migration restored. Found the error was in the schema where a field created in this work was already in merged code.
